### ISSUE TITLE
Visualising Normal vectors

### ISSUE THREAD

**Issue by basit-7** [OuyangJunyuan/PointCloudRenderer#2]
Hi @OuyangJunyuan, this is exception work.

 Is it possible to render normal vectors for the point cloud?

**Comment by OuyangJunyuan** [comment]
> Hi @OuyangJunyuan, this is exception work.
> 
> Is it possible to render normal vectors for the point cloud?

Of course, it can. But slight modifications should be added by yourself to it.
